test-requirements.txt

```diff
@@ -4,3 +4,4 @@ pexpect
 yamllint
 pytest-xdist
 coverage==4.5.4
+deepdiff
```

Comment:
Make sure to add `tests/integration/targets/dcnm_network/files/` to https://github.com/CiscoDevNet/ansible-dcnm/blob/develop/.gitignore